### PR TITLE
Distribute builder images

### PR DESCRIPTION
**Pull request by alvaroaleman**
I took those from core-services/prevent-image-gc/daemonset.yaml

Are there other places I should look at? Also, should we put that DaemonSet onto all clusters?

### CODE REVIEW

File: clusters/app.ci/assets/dptp-controller-manager.yaml [hunk 1]
```diff
@@ -177,6 +177,15 @@ spec:
         - --github-token-path=/etc/github/oauth
         - --testImagesDistributorOptions.imagePullSecretPath=/etc/regcred/.dockerconfigjson
         - --testImagesDistributorOptions.additional-image-stream-tag=ci/clonerefs:latest
+        - --testImagesDistributorOptions.additional-image-stream-tag=openshift/release:golang-1.10
```
Comment (stevekuznetsov) [inline clusters/app.ci/assets/dptp-controller-manager.yaml:180]: It's the `ocp/builder` that we need

Comment (alvaroaleman) [reply clusters/app.ci/assets/dptp-controller-manager.yaml:180]: There are currently 385 tags in that stream, do we need all of them? If yes I would probably add a new `additionalImageStream` option

Comment (stevekuznetsov) [reply clusters/app.ci/assets/dptp-controller-manager.yaml:180]:
```
$ oc --context api.ci get is --namespace ocp builder -o "jsonpath={.status.tags[*].tag}"
golang-1.10
golang-1.11
golang-1.12
golang-1.13
rhel-7-golang-1.11
rhel-7-golang-1.12
rhel-7-golang-1.13
rhel-8-golang-1.11
rhel-8-golang-1.12
rhel-golang-1.11
rhel-golang-1.12
rhel-golang-1.13
```

?

However, not a bad idea

Comment (alvaroaleman) [reply clusters/app.ci/assets/dptp-controller-manager.yaml:180]: Ah, did a `k get imagestreamtag -n ocp |grep builder|wc -l` I guess that caught tags from other streams as well

**Pull request merged**